roles/kubernetes/kubeadm/tasks/main.yml

```diff
@@ -71,7 +71,7 @@
     owner: "root"
     mode: "0644"
   when:
-    - not is_kube_master
+    - ('kube_control_plane' not in group_names)
```

Comment:
same for kube_control_plane on each occurence.

Reply:
Same reason as above - if I do not do something about the quote, the YAML parser will fail when interpreting the line.